src/app/core/filter/filters/filters.ts

```diff
@@ -78,13 +78,15 @@ export class SelectableFilter<T extends Entity> extends Filter<T> {
     valuesToMatchAsOptions: (string | number)[],
     attributeName: string,
   ): FilterSelectionOption<T>[] {
+    let keys = new Set();
     return valuesToMatchAsOptions
       .filter((k) => !!k)
       .map((k) => ({
         key: k.toString().toLowerCase(),
         label: k.toString(),
         filter: { [attributeName]: k } as DataFilter<T>,
-      }));
+      }))
+      .filter((value) => !keys.has(value.key) && keys.add(value.key));
```

Comment:
wouldn't it be slightly more readable to just convert the result to a Set to get unique values rather than this filter function?
`return [...new Set(resultWithDuplicates)];`
https://dev.to/clairecodes/how-to-create-an-array-of-unique-values-in-javascript-using-sets-5dg6

Reply:
oh, you are probably right! :blush: 
```suggestion
      // remove duplicates:
      .filter((value) => !keys.has(value.key) && keys.add(value.key));
```

(this is probably also possible without the Set as helper with somethign like `filter((value, index, array) => array.find(e => e.key) === index);`  but either one works)